docs/proof-of-contribution.md

```diff
@@ -0,0 +1,96 @@
+# Proof of Contribution
+
+The consensus algorithm used to secure and distribute the gno.land chain.
+
+Meta issue: [#918](https://github.com/gnolang/gno/issues/918).
+
+Presentation: https://github.com/gnolang/workshops/tree/main/presentations/2023-06-06--buidl-asia--manfred.
+
+## Main Concepts
+
+- Validator set determined by `worxDAO`, a DAO serving as the authority.
+- Governance and distribution managed through smart contracts.
+- Chain monitors contract changes (e.g., `valset`) to configure `Tendermint2`.
+- No staking involved in consensus.
+- Chain fees distributed to contributors and validators, not stakers.
+- Chain fees accumulated in a contract-managed bucket for efficient distribution.
+- Validators likely have equal power (1).
+- Validators do not vote like in PoS, but may participate in dedicated governance topics, maybe.
+
+## High-level schema
+
+            ____                   ____         ____                    __       _ __          __  _
```

Comment:
Can we use images for graphs? It does not render properly on all screen sizes:
![screenshot-github com-2023 07 07-09_52_38](https://github.com/gnolang/gno/assets/1196465/f955625a-4ad6-4cbe-8492-46fea7540f34)

Reply:
Yes, we can if they can be generated from something (graphviz?).

Reply:
> Yes, we can if they can be generated from something (graphviz?).


Yes! I usually use Graphviz DOT language to generate the image and keep that code as an HTML comment to make it easier to modify for the next one: https://edotor.net/